js-miniapp-sdk/scripts/docs.sh

```diff
@@ -17,7 +17,7 @@ version: "$PACKAGE_VERSION"
 cat <<< "$contents" > "$FILE_VERSION_MD"
 
 # Generate docs
-npx typedoc --includeVersion --out $DIR_DOCS/api src --plugin typedoc-plugin-markdown --stripInternal --readme none --mode file
+npx typedoc --out $DIR_DOCS/api src --options typedoc.json
```

Comment:
when I generated the doc locally, I still see the version of js-miniapp-bridge 1.1.0

Reply:
Oh strange, did you use `yarn sdk docs`? Also, did you delete the `publishableDocs` folder before running it? I think sometimes it doesn't generate correctly if you don't delete the folder first.

I checked on the CircleCI build and it looks like the name is correct in the Artifacts:
https://app.circleci.com/pipelines/github/corycaywood/js-miniapp/123/workflows/a3d0937f-6160-4c28-8dd1-26552fa46960/jobs/307/artifacts

Reply:
you're right, _delete the publishableDocs folder before running_ works.